catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/works/v2/ApiV2WorksTest.scala

```diff
@@ -11,7 +11,7 @@ class ApiV2WorksTest extends ApiWorksTestBase {
 
   it("returns a list of works") {
     withV2Api {
-      case (apiPrefix, indexName, itemType, server: EmbeddedHttpServer) =>
+      case (apiPrefix, _, indexName, itemType, server: EmbeddedHttpServer) =>
```

Comment:
Let’s fix it in a separate patch, but this seems like quite a fragile way to define the fixtures. Would be easy for somebody to get the wrong index by accident.

Reply:
I agree this is fragile. If i'm going to add another test case I might as well rename these.